src/api_server/src/parsed_request.rs

```diff
@@ -59,6 +59,9 @@ impl ParsedRequest {
         match (request.method(), path, request.body.as_ref()) {
             (Method::Get, "", None) => parse_get_instance_info(),
             (Method::Get, "balloon", None) => parse_get_balloon(path_tokens.get(1)),
+            (Method::Get, "export-vm-config", None) => {
+                Ok(ParsedRequest::new_sync(VmmAction::GetFullVmConfig))
```

Comment:
Nit: GetVmConfig

Reply:
`src/vmm/src/vmm_config/machine_config.rs` defines `VmConfig` struct which is also what we call `MachineConfig` (cpu and mem config).

I defined the complete/full VM config retrieval action as `GetFullVmConfig` to avoid confusion between the two..